### PR TITLE
welcome screen v1.0

### PR DESCRIPTION
**Pull request by cezaraugusto**

### CODE REVIEW

File: src/theme/brave-default.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ const theme: ITheme = {
     brandBatInteracting: colors.blurple500,
     brandBatActive: colors.blurple200,
     // regular controls
-    defaultControl: colors.grey800,
-    defaultControlInteracting: colors.black,
+    defaultControl: colors.grey500,
```
Comment (cezaraugusto) [inline src/theme/brave-default.ts]: cc @rossmoody changes in the theme applies for all select boxes in our app.
same as above just checking this is what we want

File: src/components/buttonsIndicators/button/style.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ const StyledButton = styled<Props, 'button'>('button')`
   border-radius: ${largeMediumSmall('24px', '20px', '16px', '28px')};
   width: ${p => p.size === 'call-to-action' ? '100%' : 'auto'};
   min-width: ${largeMediumSmall('116px', '104px', '88px', '235px')};
-  padding: ${largeMediumSmall('14px 15px', '11px 10px', '7px 10px', '19px 15px')};
+  padding: ${largeMediumSmall('14px 18px', '11px 12px', '7px 12px', '19px 15px')};
```
Comment (cezaraugusto) [inline src/components/buttonsIndicators/button/style.ts]: cc @rossmoody this is a shared component and changes here applies for all select boxes in our app.
just checking this is what we want

File: src/components/buttonsIndicators/button/style.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@ const StyledButton = styled<Props, 'button'>('button')`
   border-radius: ${largeMediumSmall('24px', '20px', '16px', '28px')};
   width: ${p => p.size === 'call-to-action' ? '100%' : 'auto'};
   min-width: ${largeMediumSmall('116px', '104px', '88px', '235px')};
-  padding: ${largeMediumSmall('14px 18px', '11px 12px', '7px 12px', '19px 15px')};
+  padding: ${largeMediumSmall('14px 15px', '11px 10px', '7px 10px', '19px 15px')};
```
Comment (cezaraugusto) [inline src/components/buttonsIndicators/button/style.ts]: cc @rossmoody this is a shared component and changes here applies for all select boxes in our app.
just checking this is what we want

**Comment by cezaraugusto** [comment]
@rossmoody pls see the above. affected components below. changed parts are buttons, select boxes and colors as they were what changed in this PR:

* button
* select
* grantComplete
* grantError
* modalBackupRestore
* walletPanel
* welcomePage for rewards

File: src/components/formControls/select/style.ts [hunk 4]
```diff
@@ -52,7 +53,7 @@ export const StyledWrapper = styled<StyleProps, 'div'>('div')`
 export const StyledSelectWrapper = styled<StyleProps, 'div'>('div')`
   position: relative;
   outline: 0;
-  font-family: Poppins, sans-serif;
+  font-family: Muli, sans-serif;
```
[the remark this answers is not in the document]
Comment (rossmoody) [reply src/components/formControls/select/style.ts]: confirmed.

File: src/theme/brave-default.ts (hunk 1 above)
Comment (rossmoody) [reply src/theme/brave-default.ts]: we can dismiss this change.

File: src/components/buttonsIndicators/button/style.ts (hunk 2 above)
Comment (rossmoody) [reply src/components/buttonsIndicators/button/style.ts]: this can be dismissed

File: src/components/buttonsIndicators/button/style.ts (hunk 3 above)
Comment (rossmoody) [reply src/components/buttonsIndicators/button/style.ts]: this can be dismissed and addressed more thoughtfully in it's own update specific to buttons.

**Comment by NejcZdovc** [comment]
@cezaraugusto please remove package-lock

**Pull request merged**